src/com/goyeau/moulin/ScalaModule.test.scala

```diff
@@ -63,10 +59,6 @@ class ScalaModuleTest extends FunSuite:
   test("test should run the Scala tests"):
     lib.test()
 
-  test("test should fail on module with no tests"):
-    intercept[SubprocessException]:
-      app.test()
```

Comment:
Removed since scala-cli runs `sys.exit(...)` and that kills the entire process, including the test itself.